app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/uploadsupportingdocs/has_supporting_docs_to_upload.scala.html

```diff
@@ -0,0 +1,64 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.config.ViewConfig
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.returns.uploadsupportingdocs.routes
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.actions.RequestWithSessionData
+
+@this(
+    mainTemplate: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.main_template,
+    pageHeading: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.page_heading,
+    submitButton: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.submit_button,
+    errorSummary: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.error_summary,
+    formWrapper: uk.gov.hmrc.play.views.html.helpers.FormWithCSRF,
+    yesNo: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.yes_no,
+    caption: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.caption,
+    returnToSummaryLink: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.return_to_summary_link,
+    backLinkComponent: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.back_link
+)
+
+@(form: Form[Boolean], backLink: Call)(implicit request:RequestWithSessionData[_], messages:Messages, appConfig: ViewConfig)
+
+@key = @{"hasSupportingDocsToUpload"}
+@title = @{messages(s"$key.title")}
+@hasErrors = @{form.hasErrors || form.hasGlobalErrors}
+
+@mainTemplate(title = title, userType = request.userType, hasErrors = hasErrors) {
+
+@backLinkComponent(backLink)
+
+@if(hasErrors) {
+@errorSummary(form)
+}
+
+@formWrapper(routes.UploadSupportingDocumentsController.hasSupportingDocsToUpload(), 'novalidate -> "novalidate") {
+
+    @yesNo(
+        fieldId = key,
+        label = pageHeading(title),
+        labelAsHeading = true,
+        errorKey = form.error(key).map(e => e.message),
+        hasErrors = hasErrors,
+        selected = form.value,
+        helpText = None,
```

Comment:
we're missing a caption and help text for this page

Reply:
looking at the spec - it looks like you're probably going to do this page properly in another PR. Is that right?

Reply:
That's not what this PR is for - that's a separate ticket - as I mentioned in the PR description it is just to setup the basic task list item mechanism